docs/c8ql/got-tutorial/import-and-merge.md

```diff
@@ -0,0 +1,247 @@
+---
+sidebar_position: 40
+title: Part 4 - Import Traits and Merge
+---
+
+This section explains how to use queries to combine information in collections.
+
+## References to Other Document Collections
+
+The character data you imported has an attribute _traits_ for each character, which is an array of strings. It does not store character features directly, however. Instead, it lists letters that represent traits. For example, Ned Stark has five traits.
+
+```json
+{
+    "name": "Ned",
+    ...
+    "traits": ["A","H","C","N","P"]
+}
+```
+
+The idea here is that _traits_ is supposed to store documents keys of another collection, which you can use to resolve the letters to labels such as "strong". By using another collection for the actual traits, you can easily query for all existing traits later on and store labels in multiple languages for instance in a central place. If you embedded traits directly, it becomes unmanageable.
```

Comment:
```suggestion
The idea here is that _traits_ is supposed to store documents keys of another collection, which you can use to resolve the letters to labels such as `strong`. By using another collection for the actual traits, you can easily query for all existing traits later on and store labels in multiple languages for instance in a central place. If you embedded traits directly, it becomes unmanageable.
```

Maybe use code box tag for individual traits?  Not sure if it's really helpful, up to you.